metrotranscom/doorway
# 44/align standard table

# Pull Request Template

## Issue Overview

This PR addresses [#44](https://github.com/bloom-housing/Bloom-Doorway/issues/44)


## Description

- It adds doorway specific class to StandardTable 
- For GroupedTable just provided temporary solution, as it will be probably changed to StandardTable in [this task](https://github.com/bloom-housing/Bloom-Doorway/issues/46)
- changed link:uic script in package.json so it works for doorway (if that should stay as it was just let me know)
- `StandardTable.stories.tsx` got mocks in it, and they are imported in few places in doorway-uic so i just left it, as it will be rather problematic to remove it.

## How Can This Be Tested/Reviewed?

Check if every StandardTable works as expected (mostly check if they have border radius)

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
